src/main/java/uk/gov/hmcts/reform/jobscheduler/services/jobs/JobsService.java

```diff
@@ -60,4 +66,18 @@ public void delete(String id, String serviceName) {
         }
     }
 
+    public List<Job> getAll(String serviceName) {
+        return getFromScheduler(scheduler::getJobKeys, GroupMatcher.jobGroupEquals(serviceName))
+            .stream()
+            .map(jobKey -> {
+                JobDetail jobDetail = getFromScheduler(scheduler::getJobDetail, jobKey);
+
+                return new Job(
```

Comment:
we'll need a separate model for reads imho

Reply:
(with ID for example)

Reply:
Ah. forgot the ID